src/components/Person/Person.jsx

```diff
@@ -1 +1,23 @@
-// export const Person = ({ person }) => ();
+export const Person = ({ person }) => {
+  const { name, age, sex, isMarried, partnerName } = person;
+
+  return (
+    <section className="Person">
+      <h2 className="Person__name">
+        {`My name is ${name}`}
+      </h2>
+      {age && (
+      <p className="Person__age">
+        {`I am ${age}`}
+      </p>
+      )}
+      {isMarried ? (
+        <p className="Person__partner">
+          {`${partnerName} is my ${sex === 'm' ? 'wife' : 'husband'}`}
+        </p>
```

Comment:
using nested ternary operator is a bad practice, it's better to move it to the variable